recipes/python-awips/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2017, Unidata Python AWIPS Developers.
```

Comment:
If you are building from the github archive tarball rather than pypi, the `license_file` should be included in the source distribution and you don't need to manually include it here. Please note that the file in the repo is `LICENSE` rather than `LICENSE.txt`

Reply:
Thanks, wasn't sure. I think it was the file extension mistake that led me to believe `conda-build python-awips` was failing without the license file including the PR.